lib/src/flips.dart

```diff
@@ -10,9 +8,9 @@ class Flips {
   const Flips.defaults() : this(false, false, false);
 
   Flips copyWith({
-    bool horizontally,
-    bool vertically,
-    bool diagonally,
+    bool? horizontally,
+    bool? vertically,
+    bool? diagonally,
```

Comment:
Couldn't these be non-nullable and have default values?

Reply:
I'm not sure, but bool types could then have three values, true, false, and null.